client/src/components/auth/Register.js

```diff
@@ -1,83 +1,93 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types';
-import './Register.css'
```

Comment:
This Register.css should be imported to Register.js so that the boarder that I created on the Sign up page reflects the css change :)